### ISSUE TITLE
error on rvm install after upgrade

### ISSUE THREAD

**Issue by brianstorti** [wayneeseguin/rvm#3252]
I got an error when I tried to run `rvm install` and a newer version was found. I just tried running it again and it worked fine, just wanted to let you know about the problem.

```shell
$ cat .rvmrc
rvm_autoupdate_flag=2
```

```shell
$ rvm install 2.2.0
Found old RVM 1.26.4 - updating.
Downloading https://get.rvm.io
No GPG software exists to validate rvm-installer, skipping.
Downloading https://github.com/wayneeseguin/rvm/archive/master.tar.gz

Upgrading the RVM installation in /Users/brianstorti/.rvm/
    RVM PATH line found in /Users/brianstorti/.bashrc /Users/brianstorti/.zshrc.
    RVM sourcing line found in /Users/brianstorti/.bash_profile /Users/brianstorti/.zlogin.
Upgrade of RVM in /Users/brianstorti/.rvm/ is complete.

# Brian Thomas Storti,
#
#   Thank you for using RVM!
#   We sincerely hope that RVM helps to make your life easier and more enjoyable!!!
#
# ~Wayne, Michal & team.

In case of problems: http://rvm.io/help and https://twitter.com/rvm_io

Upgrade Notes:

  * No new notes to display.

RVM reloaded!
find: ftsopen: No such file or directory
__rvm_run_wrapper:source:7: no such file or directory: /base
__rvm_run_wrapper:source:8: no such file or directory: /manage
```

Thanks for your work!

**Comment by mpapis** [comment]
try:
```bash
rvm-restart
rvm get head
rvm-restart
echo 1.26.4 > $rvm_path/VERSION
rvm install 2.2.0
```
does it happen again - I remember seeing similar bug in earlier version and I think it's fixed already

**Comment by mpapis** [comment]
try again ^

**Comment by brianstorti** [comment]
It worked :)
Thanks!

**Comment by andywenk** [comment]
just wanted to let you know that I updated to `1.26.8` using `zsh` and the `.rvmrc` have not been executed when cd'ing into a dir. I then used `head` and had the same issue. When running `./.rvmrc` I received the message `You need to change your terminal emulator preferences to allow login shell.` and so on. Installing `head` again some minutes ago fixed the problem.

Thanks :+1:

**Comment by mpapis** [comment]
yep fixed via #3250
